test/e2e/specs/basic_spec.js

```diff
@@ -72,6 +72,12 @@ describe('Basic flows', function () {
   });
 
   it('has the style from config.colors', function () {
+    /*global browserName*/
+    // In IE, primaryButton.getCssValue('background') is empty. Expectation on line 96 fails
```

Comment:
do u know why ?

Reply:
I haven't been able to figure out why. But it consistently fails. 
I can create a JIRA for it and add it in the comment here.

Reply:
Update: I see this thread which explains why this doesn't work on IE - https://stackoverflow.com/questions/11503709/style-attribute-issue-with-internet-explorer